Makefile

```diff
@@ -136,6 +136,7 @@ generate-binapi: get-binapi-generators
 	cd plugins/vpp/binapi/vxlan && pkgreflect
 	@echo "=> applying fix patch"
 	patch -p1 -i plugins/vpp/binapi/fixapi.patch
+	patch -p1 -i plugins/vpp/binapi/fixapi-sr.patch
```

Comment:
We should probably just use `*.patch` here to avoid future additions and support splitting patches into files.

Reply:
I agree, but that code that you commited didn't do the trick in this case. I changed it to "find -exec" version, but with the same goal. See the fix in 3543e48.